cclient/serialization/json/get_balances.c

```diff
@@ -33,22 +33,94 @@ retcode_t json_get_balances_serialize_request(
 
   cJSON_AddItemToObject(json_root, "threshold",
                         cJSON_CreateNumber(obj->threshold));
+  if (obj->tips) {
+    ret = hash243_queue_to_json_array(obj->tips, json_root, "tips");
+    if (ret != RC_OK) {
+      goto err;
+    }
+  }
+
+  json_text = cJSON_PrintUnformatted(json_root);
+  if (json_text) {
+    len = strlen(json_text);
+    ret = char_buffer_allocate(out, len);
+    if (ret == RC_OK) {
+      strncpy(out->data, json_text, len);
+    }
+  }
+
+  return ret;
```

Comment:
remove, the label below will take care of this, also, this return will create a memleak